backend/tests/Feature/Import/DryRunImportTest.php

```
@@ -0,0 +1,341 @@
+    use RefreshDatabase;
+
+    private User $admin;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Storage::fake('local');
+
+        // The test database pre-seeds roles, so never plain create().
+        $this->admin = User::factory()->create();
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\User::factory().


(staticMethod.notFound)